src/screens/CarBrowser.tsx

```diff
@@ -1,12 +1,34 @@
-import React from "react";
+import React, {useContext} from "react";
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, View, Button } from 'react-native';
-import { Text } from '../components/atoms/Text/Text';
+import { CarCard } from "../components/molecules/CarCard";
+import { View, StyleSheet, Text, TouchableOpacity, ScrollView } from "react-native";
+import { DefaultGradient } from "../components/molecules/DefaultGradient";
+import { Ionicons } from '@expo/vector-icons';
+import { CarsContext } from "../components/molecules/CarsContext";
 
 export function CarBrowser({ navigation }: { navigation: any }) {
```

Comment:
I sadly forgot a point, but this parameter list should be updated with the new format. 

The page should be setup in `App.tsx` like Home screen (I believe it already is though), as well as having the parameterlist updated. Please see Home screen on how to do this.

The listed cars should also be filtered by the location choosen on the Home screen (can be accessed through navigation route param list)

Reply:
Is fixed with location filtering